ground/src/main/java/com/google/android/ground/util/Debug.kt

```diff
@@ -40,3 +42,5 @@ object Debug {
       null
     }
 }
+
+fun Throwable.priority() = if (this is FirebaseNetworkException) Log.DEBUG else Log.ERROR
```

Comment:
How about adding a check [here](https://github.com/google/ground-android/blob/e2c703377ab69d81f755720693c326879b943bde/ground/src/main/java/com/google/android/ground/GroundApplication.kt#L63) and log to console?

Then we can continue to use `Timber.e` normally in all classes